config/routes.rb

```diff
@@ -1,3 +1,17 @@
 Rails.application.routes.draw do
+  resources :posts
+  
+  get 'posts/index'
+  get 'posts/show'
+  get 'posts/new'
+  get 'posts/edit'
+  post 'posts/new', to: 'post#create'
```

Comment:
Is this one really necessary? I understand that this is the natural flow of Rails and there's no need to specify it. 

[`OPTIONAL`]

Reply:
Done